crates/storage_models/src/address.rs

```diff
@@ -88,6 +88,28 @@ pub struct AddressUpdateInternal {
     modified_at: PrimitiveDateTime,
 }
 
+impl AddressUpdateInternal {
+    pub fn create_address(self, source: Address) -> Address {
+        let _address_update_internal = self;
```

Comment:
You can either remove the underscore prefix, or use `self` directly when accessing the fields.

Reply:
done